File: src/test/java/com/example/screenplay/question/NumberOfReturnedPosts.java
package com.example.screenplay.question;

import net.serenitybdd.screenplay.Actor;
import net.serenitybdd.screenplay.rest.questions.LastResponse;

public class NumberOfReturnedPosts extends QuestionWithDefaultSubject<Integer> {

  @Override
  public Integer answeredBy(Actor actor) {
    return actor.asksFor(new LastResponse()).jsonPath().getList("").size();
  }
}

File: src/test/kotlin/com/example/screenplay/question/NumberOfReturnedPosts.kt
package com.example.screenplay.question

import net.serenitybdd.screenplay.Actor
import net.serenitybdd.screenplay.rest.questions.LastResponse

class NumberOfReturnedPosts : QuestionWithDefaultSubject<Int>() {
  override fun answeredBy(actor: Actor): Int {
    return actor.asksFor(LastResponse()).jsonPath().getList<Any>("").size
  }
}